packages/plugin-baby/README.md

```diff
@@ -0,0 +1,52 @@
+# Plugin helpers
+
+Set of helpers based on the inversify.js that enable implementing a plugin architecture in your app.
+
+## EventedContainer
+
+Tiny wrapper on top of inversify.js Container that allow you to subscribe to the bind, unbind and rebind events.
+
+## ExtensionProvider
+
+Generic Extension provider that holds a container and a service identifier. You can bind your concrete implementations using `bindExtensionProvider` function like so:
+
+```js
+const MyServiceExtension = Symbol("MyServiceExtension");
+
+bindExtensionProvider(container.bind, MyServiceExtension)
+```
+
+then you can inject named provider in your service
+
+```js
+@injectable
+class MyService {
+  constructor(
+    @inject(ExtensionProvider) @named(MyServiceExtension) extensionProvider: ExtensionProvider
+  ) {
+    this._extensionProvider = extensionProvider;
+  }
+}
+```
+
+having `MyService` you can go ahead and bind `MyServiceExtension` to your extensions and the `ExtensionProvider` will make sure `MyService` gets them.
+
+```js
+container.bind(MyServiceExtension).to(RandomExtension)
+```
+
+## Application
+
+`Application` service is a generic application that you can use or you can use your own. `Application` exposes only one public method `start` that will re-initialize all `ApplicationExtension`s bound to the container right after they were bound.
```

Comment:
I would prefer to have the last sentence in two parts. Like `method start. start will ...` Also I don't understand from the sentence what start is doing.